tests/Integration/Api/GetHookStatusTest.php

```diff
@@ -45,18 +45,18 @@ public function testGetHookStatus(): void
         $activeHook->add();
 
         $bearerToken = $this->getBearerToken();
-        $response = static::createClient()->request('GET', '/new-api/hook-status/' . $inactiveHook->id, ['auth_bearer' => $bearerToken]);
+        $response = static::createClient()->request('GET', '/admin-dev/new-api/hook-status/' . (int) $inactiveHook->id, ['auth_bearer' => $bearerToken]);
```

Comment:
Ah I didn't realize integration tests were already present However the namespace should be changed to `ApiPlatform` as well

Reply:
I think we should add a custom TestCase that will offer some helper methods to test the APIs more easily, but it should be done in another PR as well

Reply:
There's an api test case for api platform ;)

Reply:
Ah but it's already used!